translations/monero-core_fr.ts

```diff
@@ -150,7 +150,7 @@
     <message>
         <location filename="../pages/History.qml" line="70"/>
         <source> selected: </source>
-        <translation> séléctionné : </translation>
+        <translation> sélectionné&nbsp;: </translation>
```

Comment:
OK. Cool. We haven't really tested the translations yet, so i'm not sure how the urlencoded strings will be treated.